web/app.yaml

```diff
@@ -1,44 +1,39 @@
-runtime: python27
-api_version: 1
-threadsafe: yes
+runtime: python37
 default_expiration: "0d 3h"
 
-builtins:
-- remote_api: on
-
 handlers:
-- url: /favicon\.ico
-  static_files: favicon.ico
-  upload: favicon\.ico
-
-- url: /pictures
-  static_dir: pictures
-
-- url: /scripts/assets/FontManifest.json
-  static_files: scripts/assets/FontManifest.json
-  upload: scripts/assets/FontManifest\.json
-  http_headers:
-    Access-Control-Allow-Origin: "*"
-
-- url: /scripts
-  static_dir: scripts
-
-- url: /styles
-  static_dir: styles
-
-- url: .*
-  script: main.app
-# Enable for version == prod.
-  secure: always
-
-libraries:
-- name: webapp2
-  version: "2.5.2"
-
-skip_files:
```

Comment:
Why drop the skip files?

Reply:
It's not listed as an option in the config guide for 3.7:

https://cloud.google.com/appengine/docs/standard/python3/config/appref

I assumed it had been deprecated in favor of .gloudignore, which we could also consider using.

Reply:
sgtm